docs/example/playground.md

```
@@ -0,0 +1,82 @@
+
+</template>
+
+<template #playground="{ operationId, openapi }">
+
+<div class="OAContent has-sidebar vp-doc">
+
+<div class="w-full max-w-xs mt-2 mx-auto text-center">
+  <label for="spec-select" class="block text-sm font-medium text-[var(--vp-c-text-1)]">
+    Select OpenAPI Specification
+  </label>
+  <select 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)